RELEASE-NOTES.md

```diff
@@ -1,5 +1,9 @@
 # Wikibase DataModel release notes
 
+## Version 5.0.3 (alpha)
```

Comment:
Performance issues aren't bugs, bugs are wrong functionality.

> A bug fix is defined as an internal change that fixes incorrect behavior.

Behaviour of `copy` is totally correct, it's not buggy at all.